include/raft.h

```diff
@@ -79,6 +79,13 @@ typedef unsigned long long raft_index;
  */
 typedef unsigned long long raft_time;
 
+/**
+ * Hold the features a raft node is capable of.
+ * This typedef must always be 8 bytes wide in order not to mess with ABI
+ * compatibility between raft versions.
```

Comment:
It's a tad unfortunate that the type we use is dictated not strictly by its intent/goal, but by an implementation detail about padding/ABI compatibility. I have an idea about this, perhaps I can submit a follow-up PR to see what people think. Not blocking, just wanted to point that out.

Reply:
Thanks. At this point what do you think of also converting this typedef from `uint64_t` to `unsigned long long`, for consistency with other types and to separate encoding and model concerns?

I voluteer for contextually improving `uv_encoding.c` to add boundary/validity checks, something we need to anyways since they are not there at the moment.

Reply:
I kind of like the fixed width types, would prefer to keep as-is, if we ever need to store it in one of the reserved fields it also already has the desired size.